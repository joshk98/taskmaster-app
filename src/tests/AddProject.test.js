import React from "react";
import { render } from "@testing-library/react";
import AddProject from "../components/AddProject";

describe("AddProject", () => {
  it("renders correctly", () => {
    const { asFragment } = render(<AddProject />);

    expect(asFragment()).toMatchSnapshot();
  });
});
