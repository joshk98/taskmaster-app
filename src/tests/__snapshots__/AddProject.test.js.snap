// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddProject renders correctly 1`] = `
<DocumentFragment>
  <div>
    AddProject
  </div>
</DocumentFragment>
`;
